homeassistant/components/zwave_js/sensor.py

```diff
@@ -295,3 +310,60 @@ def extra_state_attributes(self) -> dict[str, str] | None:
             return None
         # add the value's int value as property for multi-value (list) items
         return {"value": self.info.primary_value.value}
+
+
+class ZWaveNodeStatusSensor(SensorEntity):
+    """Representation of a node status sensor."""
+
+    def __init__(
+        self, config_entry: ConfigEntry, client: ZwaveClient, node: ZwaveNode
+    ) -> None:
+        """Initialize a generic Z-Wave device entity."""
+        self.config_entry = config_entry
+        self.client = client
+        self.node = node
+        name: str = (
+            self.node.name
+            or self.node.device_config.description
+            or f"Node {self.node.node_id}"
+        )
+        # Entity class attributes
+        self._attr_name = f"{name}: Node Status"
+        self._attr_unique_id = (
+            f"{self.client.driver.controller.home_id}.{node.node_id}.node_status"
+        )
+        self._attr_entity_registry_enabled_default = False
+        self._attr_should_poll = False
```

Comment:
These are static, so can be moved to keep the constructor smaller

But a matter of taste though 😄

Reply:
I think it's nice to have them all grouped. The others are static too, just derived

Reply:
Let's put the attributes that done need parameter input as class attributes per the suggestion. That's our current informal standard.